src/OrchardCore.Modules/OrchardCore.Settings/Drivers/DefaultSiteSettingsDisplayDriver.cs

```diff
@@ -84,7 +91,21 @@ public override async Task<IDisplayResult> UpdateAsync(ISite site, UpdateEditorC
                 }
             }
 
-            return Edit(site);
+            return await EditAsync(site, context);
+        }
+
+        private static void PopulateProperties(ISite site, SiteSettingsViewModel model)
```

Comment:
Just one question here, How we can make this extensible? Or we should subclass the `DefaultSettingsDisplayDriver`?

Reply:
You can create a new display driver using "general" as the groupId. Then you can place your custom settings into one of the 3 existing tabs or a new custom tab.

Reply:
I need to try this, but it's a good addition

Reply:
added an example in the description on the PR.